lcli/src/main.rs

```diff
@@ -449,6 +450,22 @@ fn main() {
                             "If present, a interop-style genesis.ssz file will be generated.",
                         ),
                 )
+                .arg(
+                    Arg::with_name("derived-genesis-state")
+                        .long("derived-genesis-state")
+                        .takes_value(false)
+                        .help(
+                            "If present, a genesis.ssz file will be generated with keys generated from a given mnemonic.",
+                        ),
+                )
+                .arg(
+                    Arg::with_name("mnemonics-phrase")
+                        .long("mnemonics-phrase")
+                        .value_name("MNEMONICS_PHRASE")
```

Comment:
minor - I think the singular form `mnemonic phrase(s)` is a bit more common, but I could be wrong here

Reply:
Agree, will make the change.